src/sentry/integrations/metric_alerts.py

```diff
@@ -72,13 +70,7 @@ def incident_attachment_info(incident, new_status: IncidentStatus, metric_value=
     title = f"{status}: {alert_rule.name}"
 
     title_link = absolute_uri(
-        reverse(
-            "sentry-metric-alert",
-            kwargs={
-                "organization_slug": incident.organization.slug,
-                "incident_id": incident.identifier,
-            },
-        )
+        f"organizations/{incident.organization.slug}/alerts/rules/details/{incident.identifier}"
```

Comment:
Gah I think this actually had the unintended side effect of changing the url for all metric alerts instead of just the unfurl.